WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -159,7 +159,7 @@ await foreach (var offendingAlices in CheckTxoSpendStatusAsync(round, cancel).Co
 					}
 				}
 
-				if (round.InputCount < Config.MinInputCountByRound)
+				if (round.UniqueScriptsCount < Config.MinUniqueInputCountByRound)
```

Comment:
Look, this is wrong everywhere. Never use `Config` in `Arena` because the `RoundParameter` is what has to be respected and contains the real parameters. `Config` can change during the round lifetiem but `RoundParameters` cannot and it is what we committed to respect and is part of the round hash.